src/components/Modals/NewMessageModal.jsx

```diff
@@ -14,9 +14,9 @@ import Typography from '@mui/material/Typography';
 import useMediaQuery from '@mui/material/useMediaQuery';
 import { useTheme } from '@mui/material/styles';
 // Utility Imports
-import { sendMessageTTL, getMessageTTL } from '@utils';
+import { sendMessageTTL } from '@utils';
```

Comment:
You should be able to move this function into the hook as well. Are you planning to do that here, or are you saving that for a followup PR?

Reply:
Yeah, I think that would be good too. Was thinking of have them moved over in follow-up PR.